src/keypress.c

```diff
@@ -233,6 +249,5 @@ void process_keypress(int c) {
             lines[cy].ident = 0;
 
         syntaxHighlight();
-        syntaxHighlight();
```

Comment:
This double call to syntaxHighlight was intentional?

Reply:
No, maybe an automatic merge that didn't work correctly?